src/js/installation.js

```diff
@@ -7,9 +7,10 @@ const loading = document.getElementById('loading');
 const errorContainer = document.getElementById('error-container');
 const platformSelector = document.getElementById('platform-selector');
 
-global.copyClipboard = (elementSelector) => {
+global.copyPreviousSibling = (target) => {
```

Comment:
What are we actually copying, can we name the function for what it is copying?

Reply:
Agreed, will push a change here shortly.